src/universal/modules/welcome/containers/Welcome/Welcome.js

```diff
@@ -62,46 +90,55 @@ export default class WelcomeContainer extends Component {
     const teamId = shortid.generate();
     const teamMemberId = shortid.generate();
     const {user} = getAuth();
-    dispatch(setWelcomeTeam({teamName, teamId, teamMemberId}));
+    dispatch(setWelcomeTeam({teamId, teamMemberId}));
     const createTeamOptions = {
       variables: {
         newTeam: {
           id: teamId,
           name: teamName,
-          leader: {
+          leader: [{
```

Comment:
Would you be so kinds as to handle this for us? I suspect your eyes executing the branch will be valuable too...